src/lib/core/components/visualizations/implementations/MapVisualization.tsx

```diff
@@ -106,6 +107,10 @@ const MapConfig = t.intersection([
       zoomLevel: t.number,
     }),
     baseLayer: t.keyof(baseLayers),
+    mouseModeConfig: t.keyof({
```

Comment:
I would just call it `mouseMode`.

Is there any way to use the `MouseMode` type from web-components? To make it future proof (again)

Reply:
@bobular I have changed it to mouseMode. But I couldn't figure out a clear way to use that union of string literals in the MouseMode type from web-components to a io-ts style. So, I did not change that part.

Reply:
You can do this
```typescript
    mouseMode: t.keyof<Record<MouseMode, null>>({
      default: null,
      magnification: null
    })
```